src/yz_entropy_mgr.erl

```diff
@@ -547,7 +549,14 @@ update_throttle(State) ->
     end.
 
 calculate_current_load(_State) ->
-    yz_solrq:queue_total_length().
+    case yz_stat:get_stat([queue, total_length]) of
+        Num when is_integer(Num) ->
+            Num;
+        Unexpected ->
+            lager:error("Unexpected value for statistic [queue, total_length]: ~p",
```

Comment:
Not sure how frequently this is happening, but my guess would be only before the stat is initialized? If so, I'd make this a lager:debug since the end-user isn't going to care (I think).